swap_meet/clothing.py

```diff
@@ -1,2 +1,9 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self,condition=0,age=0):
+        super().__init__("Clothing",condition,age)
```

Comment:
Great job implementing the super initializer here! Furthermore, nice move inserting the proper category string `"Clothing"` just once in the super initializer and not in the child class initializer in L4. 💯